cmd/pyroscope/main.go

```diff
@@ -1,14 +1,7 @@
 package main
 
-import (
-	"github.com/fatih/color"
-
-	"github.com/pyroscope-io/pyroscope/pkg/cli"
-	"github.com/pyroscope-io/pyroscope/pkg/config"
-)
+import "github.com/pyroscope-io/pyroscope/cmd/pyroscope/command"
 
 func main() {
-	if err := cli.Start(new(config.Config)); err != nil {
-		fatalf("%s %v\n\n", color.RedString("Error:"), err)
-	}
+	command.Initialize()
 }
```

Comment:
It's essential to preserve `fatalf` behaviour. On Windows, Agent may run as a Windows Service in non-interactive mode, therefore any errors at start time should be written to Windows Event Log.